### PR TITLE
deleted one scrollup button

### PR DESCRIPTION
**Pull request by likitha-kapu**
Issues: #1789 
The screenshot of the resolved issue:
image

### Pull Request Checklist

- [X] I have added screenshots and videos to show before and after the working of my code.
- [X ] I have ensured that the screen size is set to 100% while making the video.
- [X ] I have synced the latest fork with my local repository and resolved any conflicts.
- [X ] I have mentioned the issue number which I created before making this PR .(format to mention issue number is : fixes #(issue number) ) 
- [X ] I understand that if any the above conditions are not met , my PR will not be MERGED .

### CODE REVIEW

**Comment by likitha-kapu** [comment]
@ANSHIKA-26 please checkout my pullrequest

File: bookmarks.html
Comment (ANSHIKA-26) [inline bookmarks.html:1]: why are there so many changes in the file bookmarks.html ? you were only required to delete a scroll to top button

**Review by ANSHIKA-26 — CHANGES_REQUESTED** [review]

**Comment by ANSHIKA-26** [comment]
button not present at home page

**Comment by ANSHIKA-26** [comment]
2 buttons still there in contact us.html

**Pull request closed without merging**